### ISSUE TITLE
Inaccurate snapper_items width?

### ISSUE THREAD

**Issue by tylersticka** [filamentgroup/snapper#56]
Greetings from Cloud Four, Filament Group friends! 👋

We've been using this plugin on a recent site we helped design and build. It's so much nicer than the alternatives we've tried on devices. Thank you for making it!

Recently I encountered an issue where sometimes, the `snapper_items` width will just be plain wrong. As near as I can _tell_ because it isn't consistent, if images haven't loaded yet, it's possible for `snapper_items` to have a `width` of `0%`, or some number lower than what it should be (`100%` for two images, for example).

Screenshots or it didn't happen:

<img width="705" alt="screen shot 2017-09-15 at 10 42 52 am" src="https://user-images.githubusercontent.com/69633/30498394-70288c7e-9a0b-11e7-9a31-6e6f930c42ce.png">

<img width="710" alt="screen shot 2017-09-15 at 11 45 07 am" src="https://user-images.githubusercontent.com/69633/30498395-702d697e-9a0b-11e7-8c50-e554f4a19eed.png">

Any idea what's going on? I thought it had to do with another plugin, and then maybe with some CSS customizations we'd made for vertical alignment, but removing those things didn't alleviate the problem. I double-checked that we are firing the plugin on `ready` (as per the docs).

I realize without a self-contained and consistently reproducible test case it's difficult to bug-fix, but a point in the right direction would be greatly appreciated. Thanks in advance! 🙏

**Comment by tylersticka** [comment]
Updating with my progress. This seems to happen (inconsistently) when images haven't loaded yet, but not when they've cached. I called `$(selector).snapper('updateWidths')` (which doesn't seem to be documented but I saw in the source) on window load and you see the problem for a moment but then it fixes itself.

If there's a way to tell the plugin to just assume each item will be `100%` width, that'd be awesome. Maybe there already is and I'm just missing it in the source?

Absent that, I'm gonna try moving the `updateWidths` call to image load events in an attempt to make the update less visually jarring for users.
